pkg/bloomcompactor/metrics.go

```diff
@@ -0,0 +1,92 @@
+package bloomcompactor
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const (
+	metricsNamespace = "loki_bloomcompactor"
+)
+
+type metrics struct {
+	compactionRunsStarted          prometheus.Counter
+	compactionRunsCompleted        prometheus.Counter
+	compactionRunsErred            prometheus.Counter
+	compactionRunDiscoveredTenants prometheus.Gauge
+	compactionRunSkippedTenants    prometheus.Gauge
+	compactionRunSucceededTenants  prometheus.Gauge
+	compactionRunFailedTenants     prometheus.Gauge
+	compactionRunSkippedJobs       prometheus.Gauge
+	compactionRunSucceededJobs     prometheus.Gauge
+	compactionRunFailedJobs        prometheus.Gauge
+	compactionRunInterval          prometheus.Gauge
+	compactorRunning               prometheus.Gauge
+}
+
+func newMetrics(r prometheus.Registerer) *metrics {
+	m := metrics{
+		compactionRunsStarted: promauto.With(r).NewCounter(prometheus.CounterOpts{
+			Namespace: metricsNamespace,
```

Comment:
namespace is `loki`, subsystem is `bloomcompactor`

```suggestion
			Namespace: metricsNamespace,
			Subsystem: metricsSubsystem,
```

Reply:
Done thank you!